### PR TITLE
EE-12327: Add event IDs for access code logs, add unit tests

### PR DESCRIPTION
**Pull request by guysidebottom**
Add event IDs to log events when sending a successful response to HMRC.

### CODE REVIEW

File: src/main/java/uk/gov/digital/ho/pttg/hmrc/HmrcClient.java [hunk 1]
```diff
@@ -70,7 +73,7 @@ public AccessCodeHmrc getAccessCodeFromHmrc(String totpCode) {
             }
         }
 
-        log.info("Received access code response");
+        log.info("Received access code response", value(EVENT, HMRC_ACCESS_CODE_RESPONSE_SUCCESS));
```
Comment (owainrobertsbjss) [inline src/main/java/uk/gov/digital/ho/pttg/hmrc/HmrcClient.java]: We've now got the HMRC_ACCESS_CODE_RESPONSE_SUCCESS used in two places in the code, meaning it'll look like we've got double when we search for this Event ID. We use this event to signify that the call to this service (hmrc-access-code) was successful. We need a slightly different event to signal that the call from this service to HMRC was successful.

File: src/main/resources/application.properties [hunk 2]
```diff
@@ -34,7 +34,7 @@ info.app.version=${version:0.0.1}
 #
 # HMRC endpoints
 #
-base.hmrc.url=https://test-developer.service.hmrc.gov.uk
+base.hmrc.url=http://localhost:4000
```
Comment (owainrobertsbjss) [inline src/main/resources/application.properties]: Have you checked this in by accident when testing locally with the stub?

Comment (guysidebottom) [reply src/main/resources/application.properties]: Your absolutely correct, I have.

File: src/main/java/uk/gov/digital/ho/pttg/hmrc/HmrcClient.java (hunk 1 above)
Comment (guysidebottom) [reply src/main/java/uk/gov/digital/ho/pttg/hmrc/HmrcClient.java]: Resolved this by adding a new log event: HMRC_GET_ACCESS_CODE_SUCCESS

**Pull request merged**